server/views.py

```diff
@@ -35,16 +35,26 @@ def get_mandatory_areas(request):
 
 @login_required
 def index(request):
+    now = datetime.datetime.now()
+
+    last_7am = datetime.datetime(now.year, now.month, now.day - (1 if now.hour < 7 else 0), 7, 0, 0)
+    next_7am = last_7am + datetime.timedelta(days=1)
     context = {
```

Comment:
just fixing some spacing
```suggestion
    now = datetime.datetime.now()
    last_7am = datetime.datetime(now.year, now.month, now.day - (1 if now.hour < 7 else 0), 7, 0, 0)
    next_7am = last_7am + datetime.timedelta(days=1)
    
    context = {
```